pandas/tests/extension/base/groupby.py

```diff
@@ -56,6 +56,39 @@ def test_groupby_agg_extension(self, data_for_grouping):
         result = df.groupby("A").first()
         self.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "expected",
+        [
+            pd.DataFrame(
+                {
+                    "td": {
+                        0: pd.Timedelta("0 days 01:00:00"),
+                        1: pd.Timedelta("0 days 01:15:00"),
+                        2: pd.Timedelta("0 days 01:15:00"),
+                    }
+                }
+            )
+        ],
+    )
+    def test_groupby_agg_extension_timedelta_cumsum_with_named_aggregation(
+        self, expected
+    ):
+        ts = pd.DatetimeIndex(
```

Comment:
Add the GH issue number as a comment on the first line of the test; `# GH#41720` here.

Reply:
Sure, I have added that here: d12629013a9e68981b7715bd0f53d8851f8bdb0f